src/components/text-editor/examples/text-editor-custom-triggers.tsx

```diff
@@ -247,6 +252,12 @@ export class TextEditorCustomTriggersExample {
     private handleListChange = (
         event: LimelListCustomEvent<ListItem<number>>,
     ) => {
+        if (this.insertMode === 'html') {
+            this.triggerFunction.insertHtml(event.detail.text);
```

Comment:
Can't we wrap it in html instead of just doing it for Ant-man?
```suggestion
            this.triggerFunction.insertHtml(`<strong>${event.detail.text}</strong>`);
```